README.md

```diff
@@ -3,43 +3,46 @@
-code and can be added to any web app simply by including a couple JavaScript files.
+[Firepad](http://www.firepad.io/) is an open-source, collaborative code and text editor. It is
+designed to be embedded inside larger web applications. 
 
-Visit [firepad.io](http://www.firepad.io/) for a demo as well as instructions on how to embed it
-into your application!
+## Live Demo
 
-Join our [Firepad Google Group](https://groups.google.com/forum/#!forum/firepad-io) to ask
-questions, request features, or share your Firepad apps with the community.
+Visit [firepad.io](http://demo.firepad.io/) to see a live demo of Firepad in rich text mode, or the
+[examples page](http://www.firepad.io/examples/) to see it setup for collaborative code editing.
 
+[![a screenshot of demo.firepad.io including a picture of two cats and a discussion about fonts](screenshot.png)](http://demo.firepad.io/)
 
-## Contributing
+## Setup
+Firepad uses [Firebase](https://www.firebase.com/?utm_source=firepad) as a backend, so it requires no server-side
+code. It can be added to any web app by including a few JavaScript files
 
-If you'd like to contribute to Firepad, you'll need to run the following commands to get your
-environment set up:
+	<!-- Firebase -->
+	<script src="https://cdn.firebase.com/js/client/2.0.2/firebase.js"></script>
 
-```bash
-$ git clone https://github.com/firebase/firepad.git
-$ cd firepad                # go to the geofire directory
-$ npm install -g grunt-cli  # globally install grunt task runner
-$ npm install -g bower      # globally install Bower package manager
-$ npm install               # install local npm build / test dependencies
-$ bower install             # install local JavaScript dependencies
-$ grunt watch               # watch for source file changes
-```
+	<!-- ACE and its JavaScript mode and theme files -->
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.1.3/ace.js"></script>
 
-`grunt watch` will watch for changes in the `/lib/` directory and lint, concatenate, and minify the
-source files when a change occurs. The output files are written to the `/dist/` directory.
+	<!-- Firepad -->
+	<link rel="stylesheet"
+	    href="https://cdn.firebase.com/libs/firepad/1.1.0/firepad.css" />
+	<script src="https://cdn.firebase.com/libs/firepad/1.1.0/firepad.min.js"></script>
 
-You can run the test suite by navigating to `file:///path/to/firepad/tests/index.html` or via the
-command line using `grunt test`.
+and calling an init function.
 
+	<div id="firepad"></div>
+	<script>
+	  var firepadRef = new Firebase('<FIREBASE URL>');
+	  var editor = ace.edit('firepad');
+	  var firepad = Firepad.fromACE(firepadRef, editor);
+	</script>
+    
+It supports rich text and code via [ACE](http://ace.c9.io/) or [CodeMirror](http://codemirror.net/). Check out the
+detailed setup instructions at [firepad.io/docs](http://www.firepad.io/docs).
 
-## Source Code Overview
+### What's Here
```

Comment:
I've gotten positive feedback in general from this kind of summary. Devs seem to like them, so I'm inclined to leave it.